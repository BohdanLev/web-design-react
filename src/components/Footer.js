import React, {Component} from 'react';


export default class Footer extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div>
        <footer className="page-footer font-small mdb-color dark pt-1 ">
          <div className="    container grey mb-3">
            <div className="footer-cols row">
              <ul>
                <div className="footer-text-title">Get started</div>
                <li>
                  <a href="#">Home</a>
                </li>
                <li>
                  <a href="#">Sign up</a>
                </li>
                <li>
                  <a href="#">Donwloads</a>
                </li>
              </ul>
              <ul>
                <div className="footer-text-title">About us</div>
                <li>
                  <a href="#">Company</a>
                </li>
                <li>
                  <a href="#">Information</a>
                </li>
                <li>
                  <a href="#">Contact us</a>
                </li>
              </ul>
              <ul>
                <div className="footer-text-title">Support</div>
                <li>
                  <a href="#">FAQ</a>
                </li>
                <li>
                  <a href="#">Help desk</a>
                </li>
                <li>
                  <a href="#">Forums</a>
                </li>
              </ul>
              <ul>
                <div className="footer-text-title">Legal</div>
                <li>
                  <a href="#">Terms of Service</a>
                </li>
                <li>
                  <a href="#">Terms of Usep</a>
                </li>
                <li>
                  <a href="#">Privacy Policy</a>
                </li>
              </ul>
            </div>
          </div>

          <div id="company_name" className="footer-copyright text-center bg-info">
            Â©Wanderlance 2020
          </div>
        </footer>
      </div>
    );
  }
}