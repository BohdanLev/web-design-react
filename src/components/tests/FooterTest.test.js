import React from 'react'; 
import Footer from '../Footer';
import { shallow } from 'enzyme';


describe('FooterComponent', () => {

    
      describe('Footer container initial', () => {
        const FooterContainer = (<Footer/>)

        it('renders properly', () => {
            expect(FooterContainer).toMatchSnapshot()
          })    
        
          const component = shallow(<Footer/>)

          it('company name visible', () => {
            expect(component.find('#company_name').text()).toEqual('Â©Wanderlance 2020')
          })
          
          

     })

     

})