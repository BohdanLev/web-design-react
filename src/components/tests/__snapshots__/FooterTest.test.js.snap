// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FooterComponent Footer container initial renders properly 1`] = `<Footer />`;
